README.md

```diff
@@ -1,53 +1,70 @@
-# YACS - Yet Another Course Scheduler
```

Comment:
Looks like some commits got added here by accident? Not sure why these are in the diff since they are already in the feature branch. Anyway, please remove these commits.